__tests__/actions/blockHeightActions.test.js

```diff
@@ -1,4 +1,4 @@
-import { rpc } from '@cityofzion/neon-js'
+import { rpc } from '@cityofzion/neon-js-legacy'
```

Comment:
I dont understand the point of this change, `@cityofzion/neon-js` is *already* an alias for the legacy version and `@cityofzion/neon-js-next` is an alias for the N3 version...

if this change is really important to others is should not be a part of this PR imo

Reply:
I had needed to put neon-js and neon-core in resolutions, so if I hadn't changed that, version 3.4.0 would have become version 5.3.0, which would cause a lot of bugs.

Why did I put neon-js and neon-core in resolutions ? If I have not put, there was an error when trying to run the tests, because it installed version 5.4.0 of neon-core and this version uses the new imports of Node which is the "node:crypto" and does not have a polyfill.

Reply:
I still dont think these changes are necessary at all 🤔 

To fix the issue you were seeing in the tests you just need to mock the module that is having issues... There are already examples of this

Reply:
https://github.com/CityOfZion/neon-wallet/pull/2502/files#diff-58cd020f95e2d0fcf940908c10ec93bb8cbd51085688d9ef7a1a966bb89c28ecR1 this is an example of how I mocked the blockchains services lib

Reply:
Seems like this point was solved